src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Todo-List</title>
+    <link rel="stylesheet" href="../src/index.css" />
+    <script src="https://kit.fontawesome.com/043c3d901b.js" crossorigin="anonymous"></script>
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css">
+<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.7/umd/popper.min.js"></script>
+<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"></script>
+
+  </head>
+  <body>
+    <form class="todo">
+      <div class="todo-list-title">
+        <p> Todo List</p>
+        <i id="icon" class="fas fa-sync recycle"></i>
+      </div>
+      <input type="text" class="activity" placeholder="Add to your list..." />
+      <ul class="todo-list"></ul>
+      <button type="button" class="clear bg-color border-0">Clear all completed</button>
+    </form>
```

Comment:
- Hey, please, kindly refer to the video in the project requirement to improve your design to be close to the design as much as you can. See the image of your design below:


![Screenshot from 2023-03-03 10-00-12](https://user-images.githubusercontent.com/59397159/222678843-55b3c7b7-996c-4505-9550-a2ac7c7c1691.png)